protocol/indexer/off_chain_updates/off_chain_updates.go

```diff
@@ -62,6 +62,50 @@ func CreateOrderPlaceMessage(
 	return msgsender.Message{Key: orderIdHash, Value: update}, true
 }
 
+// MustCreateOrderReplaceMessage invokes CreateOrderReplaceMessage and panics if creation was unsuccessful.
+func MustCreateOrderReplaceMessage(
```

Comment:
Can you add a unit test for this function?

Reply:
Added TestCreateOrderReplaceMessage!